### PR TITLE
docs: Add a note about extractNativeLibs usage

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: src/platforms/android/index.mdx [hunk 1]
```diff
@@ -764,3 +764,13 @@ Example of uploading all your .so files:
 sentry-cli login
 sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} build/intermediates/merged_native_libs/{buildVariant}
 ```
+
+Due to current limitations in how `sentry-native` works, it cannot yet identify native libraries loaded directly from `.apk` files.
+This functionality is [enabled by default](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs)
+when using a recent Android Gradle Plugin, and needs to be disabled with the following configuration option:
```
Comment (marandaneto) [inline src/platforms/android/index.mdx:770]:
```suggestion
when using the Android Gradle Plugin >= v3.6.0, and it needs to be enabled with the following configuration option (AndroidManifest.xml):
```

File: src/platforms/android/index.mdx [hunk 2]
```diff
@@ -764,3 +764,13 @@ Example of uploading all your .so files:
 sentry-cli login
 sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} build/intermediates/merged_native_libs/{buildVariant}
 ```
+
+Due to current limitations in how `sentry-native` works, it cannot yet identify native libraries loaded directly from `.apk` files.
+This functionality is [enabled by default](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs)
```
Comment (marandaneto) [inline src/platforms/android/index.mdx:769]:
```suggestion
This functionality is [disabled by default](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs)
```

File: src/platforms/android/index.mdx [hunk 3]
```diff
@@ -764,3 +764,13 @@ Example of uploading all your .so files:
 sentry-cli login
 sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} build/intermediates/merged_native_libs/{buildVariant}
 ```
+
+Due to current limitations in how `sentry-native` works, it cannot yet identify native libraries loaded directly from `.apk` files.
```
Comment (marandaneto) [inline src/platforms/android/index.mdx:768]: I'd still move the whole section right after the title `Integrating the NDK`

this should be the first step to be read.

**Review by marandaneto — APPROVED** [review]
LGTM, CI is not happy though.

**Comment by Swatinem** [comment]
rebased on master, hope that fixes CI

**Comment by marandaneto** [comment]
> rebased on master, hope that fixes CI

I cannot get the vercel's link `There is no Vercel account linked with this GitHub account. Sign up?`
but I think it's ready to go

**Pull request merged**